less/index.less

```diff
@@ -1,3 +1,27 @@
+/* @ Dynamic Variables for LESS */
+@bilbo-font: 'Bilbo Swash Caps', cursive;
+@standard: 62.5%;
+@color1: white;
+@color2: orange;
+@linkColor: yellow;
+
+/* Mixins */
+.img() {
+	max-width:100%;
+	alt: "";
```

Comment:
I think this `alt` property refers to something other than the HTML alt. I was working with it for a little while, but couldn't get it to work without changing the HTML diretly 🤔